konfera/tests/test_models.py

```diff
@@ -276,6 +268,25 @@ def test_clean_and_save_ticket_discount_code(self):
                                                  discount_code=self.discount_code_2)
         self.assertRaises(ValidationError, ticket_with_invalid_code.save)
 
+    def test_save_tickets_for_different_event(self):
+        title1 = random_string(128)
```

Comment:
Personally I prefer when the title is static, the assertion messages are then more readable and it takes less time to run the tests.